src/services/searchService.ts

```diff
@@ -205,23 +201,10 @@ function parseHumanDate(humanDate: string): DateValue[] {
 }
 
 async function searchLocation(
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
     searchPhrase: string
 ): Promise<LocationSearchResponse[]> {
-    try {
-        const resp = await HTTPService.get(
-            `${ENDPOINT}/search/location`,
-            {
-                query: searchPhrase,
-                limit: 4,
-            },
-            {
-                'X-Auth-Token': getToken(),
-            }
-        );
-        return resp.data.results ?? [];
-    } catch (e) {
-        logError(e, 'location search failed');
-    }
+    addLogLine(`attempting to use unimplemented search API`);
```

Comment:
we should log to sentry also, just in case

Reply:
I wasn't sure if we should use

1.    ` logError(Error('attempting to use unimplemented search API'));`

or 

2.    ` logError(Error(), 'attempting to use unimplemented search API');

I found one example in code that did #2, so I did the same.

Reply:
ya two works